x-pack/solutions/security/packages/expandable-flyout/src/store/reducers.ts

```diff
@@ -167,14 +167,30 @@ export const uiReducer = createReducer(initialUiState, (builder) => {
     state.pushVsOverlay = type;
   });
 
-  builder.addCase(setDefaultWidthsAction, (state, { payload: { right, left, preview } }) => {
-    state.defaultWidths.rightWidth = right;
-    state.defaultWidths.leftWidth = left;
-    state.defaultWidths.previewWidth = preview;
-    state.defaultWidths.rightPercentage = (right / (right + left)) * 100;
-    state.defaultWidths.leftPercentage = (left / (right + left)) * 100;
-    state.defaultWidths.previewPercentage = (right / (right + left)) * 100;
-  });
+  builder.addCase(
+    setDefaultWidthsAction,
+    (
+      state,
+      { payload: { rightOverlay, leftOverlay, previewOverlay, rightPush, leftPush, previewPush } }
+    ) => {
+      state.defaultWidths.overlay.rightWidth = rightOverlay;
+      state.defaultWidths.overlay.leftWidth = leftOverlay;
+      state.defaultWidths.overlay.previewWidth = previewOverlay;
+      state.defaultWidths.overlay.rightPercentage =
+        (rightOverlay / (rightOverlay + leftOverlay)) * 100;
+      state.defaultWidths.overlay.leftPercentage =
+        (leftOverlay / (rightOverlay + leftOverlay)) * 100;
+      state.defaultWidths.overlay.previewPercentage =
+        (rightOverlay / (rightOverlay + leftOverlay)) * 100;
```

Comment:
should this reference preview

Reply:
yup you are correct, while they are the same, it will be more clear to use `previewOverlay` here. Updated!